hackclub.com.yaml

```diff
@@ -1,4 +1,8 @@
 ---
+slss.hackclub.com:
+  - ttl: 1
+    type: CNAME
+    value: slss.
```

Comment:
This DNS record seems non-operative. What domain are you trying to point this to?

Reply:
I want the url to be slss.hackclub.com I'm sorry but I'm finding difficulty sitting it up

Reply:
What do you want slss.hackclub.com to point to? Also please see Graham's comment about providing a description for this PR.